tests/Unit/NumericalAlgorithms/LinearSolver/Test_Tags.cpp

```diff
@@ -32,4 +40,38 @@ SPECTRE_TEST_CASE("Unit.Numerical.LinearSolver.Tags",
         "LinearOrthogonalizationHistory(Tag)");
   CHECK(LinearSolver::Tags::KrylovSubspaceBasis<Tag>::name() ==
         "KrylovSubspaceBasis(Tag)");
+  CHECK(LinearSolver::Tags::ConvergenceCriteria::name() ==
+        "ConvergenceCriteria");
+
+  {
+    INFO("HasConvergedCompute");
```

Comment:
Can you add a check for `::name()` for this and the compute tag as well please? Thanks!

Reply:
The `::name()` of the simple tag `HasConverged` is tested above already. The compute tag derives from it, so should I indeed test its `::name()` too?

Reply:
Whatever you prefer